process/economics/economicsData.go

```diff
@@ -161,6 +169,10 @@ func checkValues(economics *config.EconomicsConfig) error {
 		return process.ErrNilProtocolSustainabilityAddress
 	}
 
+	if economics.FeeSettings.GasPriceModifier > 1.0 || economics.FeeSettings.GasPriceModifier < epsilon {
+		return process.ErrInvalidGasModifier
```

Comment:
maybe `fmt.Errorf("%w, max allowed %v, min allowed %v, got %v", 1.0, epsilon, process.ErrInvalidGasModifier)` ?

Reply:
this will easily faild on any constructor and checking it is easy.